src/afl-fuzz-queue.c

```diff
@@ -45,25 +45,19 @@ inline u32 select_next_queue_entry(afl_state_t *afl) {
 double compute_weight(afl_state_t *afl, struct queue_entry *q,
                       double avg_exec_us, double avg_bitmap_size) {
 
-  u32 hits;
+  double weight = 1.0;
```

Comment:
Maybe `float` is enough here? I always expect doubles to be super slow. (Gut feeling though, didn't benchmark it)

Reply:
is it? we should check in godbolt